js/local-storage.js

```diff
@@ -0,0 +1,25 @@
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const messageInput = document.getElementById('message');
+
+const saveFormData = () => {
+  const formData = {
+    name: nameInput.value,
+    email: emailInput.value,
+    message: messageInput.value,
+  };
+  localStorage.setItem('formData', JSON.stringify(formData));
```

Comment:
Let's use a more unique key, a common key might conflict with other app localStorage keys.

You can use your app url concatenated with what you are using currently.

Reply:
Done